### DISCUSSION TITLE
REQ: Spectators as hud-element

### DISCUSSION THREAD

**Discussion by blaps** [ezQuake/ezquake-source#518] [Ideas]
`hud_spectators_show 1`

As an observer/caster, I'd like to see a list of the observers on the server.
Possible configuration could be number of rows, in addition to the basic size/color etc. 
Filter out ServeMe-bot if possible?

**Comment by JeremyBakkerr** [comment]
Beim Zuschauen oder Streamen kann es entscheidend sein zu wissen, wer sich sonst noch auf dem Server befindet – sei es zur Koordination oder einfach, um den Überblick zu behalten. Ein einfaches HUD-Element, das beispielsweise über `hud_spectators_show 1` gesteuert wird, wirkt übersichtlich und passt gut zu den bestehenden Optionen. Die Möglichkeit, Zeilen, Größe und Farbe anzupassen, würde es flexibel genug für verschiedene Bildschirm-Layouts und Streams machen. KingMaker Casino https://kingmakercasino-de.de/ bietet spannendes Online-Gaming mit königlicher Atmosphäre. Das Herausfiltern von Bots wie ServeMe ist definitiv wichtig, da die Liste sonst schnell unübersichtlich wird. Insgesamt wirkt es wie eine ressourcenschonende Funktion mit echtem Nutzen, die das Gameplay in keiner Weise beeinträchtigt.

**Comment by JesseReed123** [comment]
It would be really useful to have a list of all observers on the server for casters or spectators. Ideally, the UI could allow configuration of the number of rows, along with size, color, and other display options. It would also be great [roobet casino](https://roobet-uk.co.uk/) to have a filter to exclude bots like ServeMe, so the list only shows real players. This feature would make tracking who’s watching much easier, especially during tournaments or streams, and improve the overall spectator experience without cluttering the display with unnecessary entries.

**Comment by RowanCross512** [comment]
That sounds like a really useful feature for streamers or servers with a lot of observers. Having a list of all active observers could help manage the viewership better and keep track of who’s watching. Configuring [pay by sky mobile casino](https://paybymobilecasino.net/sky-mobile-casino/) the number of rows, size, and color would give a lot of customization options for different use cases. Filtering out bots like ServeMe-bot is a smart idea too, as they clutter the list unnecessarily. If this isn’t already built into the system, it could definitely improve the observer experience and make managing them easier for the caster.
